packages/kit/src/core/adapt/index.js

```diff
@@ -7,14 +7,15 @@ import { get_utils } from './utils.js';
  * @param {import('types/internal').BuildData} build_data
  * @param {{ cwd?: string, verbose: boolean }} opts
  */
+
 export async function adapt(config, build_data, { cwd = process.cwd(), verbose }) {
 	const { name, adapt } = config.kit.adapter;
 
 	console.log(colors.bold().cyan(`\n> Using ${name}`));
 
 	const log = logger({ verbose });
 	const utils = get_utils({ cwd, config, build_data, log });
-	await adapt(utils);
+	await adapt({ utils, svelteConfig: config });
```

Comment:
trying to communicate this is not the user's config for the adapter, but in fact, the `svelte.config.js` config.